app/views/fact_check_content_review.scala.html

```diff
@@ -15,13 +15,21 @@
  *@
 
 @import models.ApprovalProcessReview
-
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 
 @this(layout: main_layout, govukTable : GovukTable, govukBackLink : GovukBackLink)
 
 @(review: ApprovalProcessReview)(implicit request: Request[_], messages: Messages)
 
+@reviewStatus() = @{
+  val totalPages = review.pages.length
+  val incompletePages = review.pages.count(p => p.status == PageReviewStatus.NotStarted)
+  incompletePages match {
+    case 0 => "Review Complete - all pages checked"
+    case _ => s"Review Incomplete - ${totalPages - incompletePages} of $totalPages pages checked"
```

Comment:
Wouldnt it be easier to count the complete pages, as thats what you are going to display, saves having to calculate the number later

Reply:
good point - I was burning the midnight oil when refactoring :)